### ISSUE TITLE
QA dropdown won't populate any data when expanding it

### ISSUE THREAD

**Issue by codejamtc** [appirio-tech/connect-app#3974]
### Expected behavior
Should display the default skillset of a QA. Like in Dev

### Actual behavior
QA dropdown won't populate any data when expanding it

### Steps to reproduce the problem

- Login as Customer
- Go to https://connect.topcoder.com/new-project/talent-as-a-service-v1.2
- Go to 'Talents' question and expand the Skill dropdown of the QA

### Screenshot/screencast

![image](https://user-images.githubusercontent.com/42398485/80605160-4faf3a80-89f8-11ea-81b9-a17d051c8bed.png)

#### Environment
- OS: Window 10
- Browser (w/version): Chrome 80.0.3987.163
- User role (client, copilot or manager): Customer 
- Account used:  `CustomerUser`

**Comment by maxceem** [comment]
@vikasrohit it looks like on PROD we don't have `QA` category skills in the API.

Only skills with the next categories are returned: `[ 'DEVELOP', 'DATA_SCIENCE', 'DESIGN', null ]`.

**Comment by vikasrohit** [comment]
Yes, it is known issue. We have got this identified when we deployed the first version of the Skills picker component.
